core/modules/wmgr/python/xrdMgr.py

```diff
@@ -94,9 +94,10 @@ def _runCmd(cmd, noexcept=True):
         else:
             raise
 
-def _restartXrootd():
+def _restartService(sName):
     """ Restart xrootd process, only if it is running already """
-    initScript = os.path.join(Config.instance().runDir, 'etc/init.d/xrootd')
+    _log.info('restarting %s', sName)
+    initScript = os.path.join(Config.instance().runDir, 'etc/init.d/%s' %sName)
```

Comment:
I prefer `'etc/init.d/' + sName` over `'etc/init.d/%s' % sName` for efficiency reasons